rubbish/src/Pos/Rubbish/Hacks.hs

```diff
@@ -12,10 +12,10 @@ import           Pos.DB     (MonadGState, gsIsBootstrapEra)
 
 -- | In order to create an 'Address' from a 'PublicKey' we need to
 -- choose suitable stake distribution. We want to pick it based on
--- whether we are currently in bootstrap era.  Light wallet doesn't
--- know current slot, so let's assume it's 0-th epoch. It's enough for
--- our current needs.
-makePubKeyAddressLWallet :: MonadGState m => PublicKey -> m Address
-makePubKeyAddressLWallet pk = do
+-- whether we are currently in bootstrap era.  Rubbisy doesn't know
```

Comment:
typo

Reply:
And if you know how to fix that we use 0th epoch, perhaps do.

Reply:
Same as in https://github.com/input-output-hk/cardano-sl/pull/1527#discussion_r137869555
I just don't want this PR to become bigger if something can be done later (that wasn't spoiled by me).

Reply:
Created https://issues.serokell.io/issue/CSL-1636